components/teamwork/sources/task-updated/task-updated.mjs

```diff
@@ -0,0 +1,24 @@
+import common from "../../common/common-sources.mjs";
+
+export default {
+  ...common,
+  type: "source",
+  key: "teamwork-task-updated",
+  name: "New Task Updated",
+  description: "Emit new event when a new task is updated",
+  version: "0.0.1",
+  methods: {
+    ...common.methods,
+    _getEventName() {
+      return "TASK.UPDATED";
+    },
+  },
+  async run(event) {
+    this._checkHmac(event.bodyRaw, event.headers["x-projects-signature"]);
+    this.$emit(event.body, {
+      id: event.body["Task.ID"],
```

Comment:
Since it's an `updated` trigger, it is necessary to add the timestamp to the id or remove the `dedupe: unique`